index.js

```diff
@@ -2,20 +2,15 @@ import 'react-native-gesture-handler';
 
 import { AppRegistry } from 'react-native';
 
-import { jobRunner } from './src/shared/lib';
+import { name as appName } from './app.json';
+import displayRemoteNotifications from './src/jobs/display-remote-notifications';
+import localization from './src/jobs/localization';
 import requestInterception from './src/jobs/request-interception';
 import responseInterception from './src/jobs/response-interception';
 import setBackgroundTask from './src/jobs/set-background-task';
-import displayRemoteNotifications from './src/jobs/display-remote-notifications';
-import localization from './src/jobs/localization';
-import App from './src/app';
```

Comment:
eslint seems to be deleting this import line when it shouldn't, but it seems to be because this file has the wrong extension – which is causing it to misinterpret the JSX contained in `HeadlessCheck` below. Renaming this file to `index.jsx` causes eslint to preserve this line (which is required for the app to work).

Reply:
Good catch Paul. I found a [similar issue](https://github.com/sweepline/eslint-plugin-unused-imports/issues/9) in the [unused-imports plugin repo](https://github.com/sweepline/eslint-plugin-unused-imports) which recommends adding these lines (which they mention in their [README](https://github.com/sweepline/eslint-plugin-unused-imports?tab=readme-ov-file#react) 🙈)

https://github.com/ChildMindInstitute/mindlogger-app-refactor/blob/fc922240c5bf945c84083f2517a5c9aeff15e329/.eslintrc.js#L92-L93

That seems to have fixed the problem